src/scripts/Nutshell.js

```diff
@@ -3,11 +3,14 @@ import "./tasks/TaskForm.js";
 import { ListFriends } from "./friends/FriendList.js"
 import { ListArticles } from "./articles/ArticleList.js"
 import "./friends/FriendForm.js"
+import { getForecast } from "./weather/WeatherProvider.js";
+import "./weather/WeatherList.js"
 
 
 export const Nutshell = () => {
     TaskList() 
     ListFriends()
     ListArticles()
+    getForecast()
```

Comment:
The convention we have been using is to call the List component from the application, rather than the provider. I recommend changing the flow of the program to match this convention